### PR TITLE
Change get to pop in compute_semivalues.

### LINKED ISSUE #330: Fix bug in compute_semi_values
The following bug occurs:

In the function compute_semi_values for beta shapley, the kwargs need to be obtained by pop and not by get.

### PR DESCRIPTION
**Pull request by kosmitive**
### Description

This PR closes #330 

### Checklist

- [x] Wrote Unit tests (if necessary)
- [x] ~Updated Documentation (if necessary)~
- [x] Updated Changelog
- [x] ~If notebooks were added/changed, added boilerplate cells are tagged with `"nbsphinx":"hidden"`~

### CODE REVIEW

**Comment by AnesBenmerzoug** [comment]
@kosmitive looks good to me. Could you please merge develop to your branch and then update the changelog?

**Comment by mdbenito** [comment]
I'm curious as to how this error manifests itself and why the tests didn't catch it. @kosmitive Maybe you could extend the tests?

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,7 +13,9 @@
   [PR #331](https://github.com/appliedAI-Initiative/pyDVL/pull/331)
 - Added alias `ApproShapley` from Castro et al. 2009 for permutation Shapley
   [PR #332](https://github.com/appliedAI-Initiative/pyDVL/pull/332)
-
+- Change get to pop i pydvl.value.semivalues.compute_semivalues method. Added
```
Comment (AnesBenmerzoug) [inline CHANGELOG.md:16]:
```suggestion
- Change `get` to `pop` in pydvl.value.semivalues.compute_semivalues method. Added
```

File: tests/value/test_semivalues.py [hunk 2]
```diff
@@ -113,3 +116,27 @@ def test_coefficients(n: int, coefficient: SVCoefficient):
...
+)
+def test_dispatch_compute_semi_values(
+    dummy_utility: Utility,
+    semi_value_mode: SemiValueMode,
+    semi_value_mode_kwargs: Dict[str, int],
+):
+    values = compute_semivalues(
+        u=dummy_utility,
+        mode=semi_value_mode,
+        done=MaxUpdates(10),
```
Comment (AnesBenmerzoug) [inline tests/value/test_semivalues.py:139]:
```suggestion
        done=MaxUpdates(1),
```

**Review by AnesBenmerzoug — APPROVED** [review]

**Pull request merged**